extensions/src/main/java/org/pentaho/platform/plugin/services/metadata/PentahoDataSourceTypeMap.java

```diff
@@ -0,0 +1,87 @@
+/*!
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
```

Comment:
[#4 fully implement changes for PentahoMetadataInformationMap.java](https://jira.pentaho.com/browse/BACKLOG-36125)